scripts/migrate_mainnet_fixtures_chainconfig.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+"""
+Script to process zkevm fixture JSON files by removing existing chain_config
+and adding a new fixed chain_config inside the block_and_witness field.
+
+This is used to migrate previously generated fixture files to the new format.
```

Comment:
Once the migration is done on existing fixtures, do we need this script after that?

Reply:
Nop, it shouldn't be needed.
I think my plan is:
- Merge this PR
- Update the witness collector VM we have so it starts to collect using the new format.
- I've to do some work on untar-ing + migrating + re-taring previously collected stuff using this script as part of that process
- After all looks good, I can delete this script in the repo.

Reply:
And hopefully we don't touch the witness generator format again :sweat_smile: